doc/rst/language/spec/data-parallelism.rst

```diff
@@ -318,11 +318,18 @@ it is subject to forall intents and all references to this field within
 the forall construct implicitly refer to the corresponding shadow
 variable.
 
-Each formal argument of a task function or iterator has the default
-intent by default.  See also :ref:`The_Default_Intent`. Note that the
+Most formal arguments of a task function or iterator have the default
+intent by default. See also :ref:`The_Default_Intent`. Note that the
```

Comment:
I prefer something like this instead of the above:

These implicit formals of task functions and iterators generally have :ref:`the default argument intent <The_Default_Intent>` by default. Note that ......

Then, start the next paragraph with:

Implicit formals of array types are an exception: they inherit ....